appveyor.yml

```diff
@@ -26,14 +25,15 @@ install:
   - set PATH=%PATH%;C:\Users\appveyor\.cargo\bin
   - set PATH=%PATH%;C:\Users\appveyor\.multirust\toolchains\nightly-x86_64-pc-windows-msvc\lib\rustlib\%TARGET%\lib
   - if NOT "%TARGET%" == "x86_64-pc-windows-msvc" rustup target add %TARGET%
+  # Set custom target in rls.toml config file
+  - if NOT "%TARGET%" == "x86_64-pc-windows-msvc" echo target = "%TARGET%" > rls.toml
```

Comment:
As I checked (`rustc -vV` output in AppVeyor logs), using default host and toolchain does that, so I set that in https://github.com/rust-lang-nursery/rls/pull/382